packages/eslint-plugin/package.json

```diff
@@ -17,9 +17,10 @@
     "typescript"
   ],
   "peerDependencies": {
-    "eslint": "^8.48.0"
+    "eslint": "^8.48.0",
+    "prettier": "^3.0.3"
   },
-  "dependencies": {
```

Comment:
this package was also bundling the prettier and everything else together.


<img width="486" alt="Screenshot 2024-02-22 at 11 11 00" src="https://github.com/FuelLabs/fuels-npm-packs/assets/7074983/235e7ae2-98ee-4988-bc11-f7aa314c7716">